packages/dear-imgui/cpp/node/hello.cpp

```
@@ -203,96 +203,96 @@ class Runner {
 
         void appendChild(const int parentId, const int childId) const {
-            m_reactImgui->QueueAppendChild(parentId, childId);
+            m_xframes->QueueAppendChild(parentId, childId);
         }
 
         [[nodiscard]] std::vector<int> getChildren(const int id) const {
-            return m_reactImgui->GetChildren(id);
+            return m_xframes->GetChildren(id);
         }
 
         [[nodiscard]] std::string getAvailableFonts() const {
```

cppcheck

[style] The function 'getAvailableFonts' is never used.

(unusedFunction)